sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/changefeed/implementation/TraceHealthMonitor.java

```diff
@@ -17,7 +17,7 @@
     public Mono<Void> inspect(HealthMonitoringRecord record) {
         return Mono.fromRunnable(() -> {
             if (record.getSeverity() == HealthMonitoringRecord.HealthSeverity.ERROR) {
-                logger.error(String.format("Unhealthiness detected in the operation %s for %s.", record.operation.name(), record.lease.getId()), record.throwable);
+                logger.error("Unhealthiness detected in the operation %s for %s.", record.operation.name(), record.lease.getId(), record.throwable);
```

Comment:
this doesn't give you what you intend. you should use slf4j place holder not `%s` please fix everywhere. slf4j will print out `%s` as is no substitution.

https://www.slf4j.org/manual.html

Reply:
Fixed

Reply:
you missed this. "%s" won't work for slf4j.

Reply:
Fixed